src/advect1d_vertical.jl

```diff
@@ -0,0 +1,253 @@
+#### This Julia file contains functions to implement 
+####  1) 1-D L94 vertical advection: advect1d_l94() (Lin et al., 1994)
+####  2) 1-D PPM vertical advection: advect1d_ppm() (Collela and Woodward, 1984)
+####  The vertical advection conserve pi*q rather than q. (Lin and Rood, 1996)
+
+#### I will upload the environment later but basically the code block below is the packages I use.
+##   -------------------------------------------------------------------------------------------
+##   ]activate EarthSciML_Advection
+##   using EarthSciData, EarthSciMLBase, GasChem,
+##   DomainSets, ModelingToolkit, DifferentialEquations, Dates
+##   using Plots, Statistics, GeoDataFrames, NCDatasets
+##   using Flux
+##   using BSON: @save
+##   using BSON: @load
+##   using Base.Threads
+##   using LoopVectorization
+##   using Tullio
+##   -------------------------------------------------------------------------------------------
+
+
+
+
+## L94 advection in 1-D
+## This is to make splitted scheme
+function advect1d_vertical_L94(s1d_in,vel_1d,nz,Δt,p)
```

Comment:
It looks like `nz` might be the length of `vel_1d`, in which case is wouldn't need to be a function argument because it could be calculated within the function body.